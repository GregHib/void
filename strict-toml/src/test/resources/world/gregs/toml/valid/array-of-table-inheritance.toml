[[fruits]]
name = "apple"

[.physical]  # subtable
color = "red"
shape = "round"

[[.varieties]]  # nested array of tables
name = "red delicious"

[[.varieties]]
name = "granny smith"


[[fruits]]
name = "banana"

[[.varieties]]
name = "plantain"