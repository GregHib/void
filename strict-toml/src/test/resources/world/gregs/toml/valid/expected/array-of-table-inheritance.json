{"fruits":[{"name":"apple","physical":{"color":"red","shape":"round"},"varieties":[{"name":"red delicious"},{"name":"granny smith"}]},{"name":"banana","varieties":[{"name":"plantain"}]}]}